src/lib/paginator/BUILD.bazel

```diff
@@ -10,9 +10,10 @@ ng_module(
   module_name = "@angular/material/paginator",
   assets = [":paginator.css"] + glob(["**/*.html"]),
   deps = [
-    "@angular//packages/common",
-    "@angular//packages/core",
-    "@rxjs",
+    "@npm//@angular/common",
+    "@npm//@angular/core",
+    "@npm//@angular/forms", # TODO(jelbourn): transitive dep via generated code
```

Comment:
For later reference: This is actually expected as we turned of the symbol re-exports (see https://github.com/angular/angular/commit/91b7152852e86311a9c880e558ae0caefc7e5e26).